### PR TITLE
localized osm wiki url

### LINKED ISSUE #2167: Redirect to localized OSM wiki pages
When you tap to "learn more about OpenStreetMap" button, it is redirecting you to English wiki page even when your device/OM language is different. Please redirect defaut language wiki page.
For example,
Turkish version of About OSM:
 https://wiki.openstreetmap.org/wiki/Tr:About

### PR DESCRIPTION
**Pull request by cyber-toad**
Fixes #2167

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -25383,3 +25383,26 @@
...
+    fr = https://wiki.openstreetmap.org/wiki/FR:À_propos_d’OpenStreetMap
+    hu = https://wiki.openstreetmap.org/wiki/Hu:Névjegy
+    it = https://wiki.openstreetmap.org/wiki/IT:About
+    ja = https://wiki.openstreetmap.org/wiki/JA:参加する
+    nl = https://wiki.openstreetmap.org/wiki/NL:Wat_is_OpenStreetMap%3F
+    pl = https://wiki.openstreetmap.org/wiki/Pl:Wstęp
+    pt = https://wiki.openstreetmap.org/wiki/Pt:Sobre_o_OpenStreetMap
+    ru = https://wiki.openstreetmap.org/wiki/RU:Присоединяйтесь!
+    sv = https://wiki.openstreetmap.org/wiki/Sv:Getting_Involved
+    tr = https://wiki.openstreetmap.org/wiki/Tr:Getting_Involved
```
Comment (biodranik) [inline data/strings/strings.txt:25407]: Why not https://wiki.openstreetmap.org/wiki/Tr:About (and for other translations too)?

The main idea of this link is to _educate_ users about "What is that OSM?". Anyone, including your granny, should be able to understand what OSM is and get the idea that this maps data is editable, and that it is a completely different project, that is used by Organic Maps.

One of the biggest issues in OM right now is that many users do not understand where does maps data come from. And they do not understand that they can easily edit it, and then get updates in OM.

File: data/strings/strings.txt [hunk 2]
```diff
@@ -25383,3 +25383,26 @@
     it = https://organicmaps.app/it/
     ru = https://organicmaps.app/ru/
     tr = https://organicmaps.app/tr/
+
+  [osm_wiki_url]
+    comment = Link to OSM wiki for About Pange
```
Comment (biodranik) [inline data/strings/strings.txt:25388]:
```suggestion
    comment = Link to OSM wiki for About Page
```

**Review by biodranik — CHANGES_REQUESTED** [review]
Can you please also fix it for iOS in iphone/Maps/UI/Help/AboutController.swift ?

**Comment by biodranik** [comment]
And also please change links to the same localized About here:

```
iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationLoginViewController.mm
104:  [self openUrl:@"https://wiki.openstreetmap.org/wiki/Main_Page"];

iphone/Maps/UI/Editor/MWMEditorNotesFooter.m
25:  [self.controller openUrl:@"https://wiki.openstreetmap.org/wiki/Main_Page"];
```

For iOS, you need to replace `@"string"` with `L("osm_wiki_url")`

By the way, maybe using osm_wiki_about_url would be more readable.

**Comment by biodranik** [comment]
Thanks! Can you please `git rebase -i` and squash 1, 3, and 4 commit into one, so there will be two commits: any logical changes and strings.txt, and regenerated strings?

**Comment by cyber-toad** [comment]
> Thanks! Can you please `git rebase -i` and squash 1, 3, and 4 commit into one, so there will be two commits: any logical changes and strings.txt, and regenerated strings?

@biodranik fixed, 2 commits are in PR now

File: data/strings/strings.txt [hunk 3]
```diff
@@ -25383,3 +25383,26 @@
     it = https://organicmaps.app/it/
     ru = https://organicmaps.app/ru/
     tr = https://organicmaps.app/tr/
+
+  [osm_wiki_about_url]
+    comment = Link to OSM wiki for About Page
```
Comment (biodranik) [inline data/strings/strings.txt:25388]: Looks like it's used in 3 different places, does it make sense to mention all of them here?

Comment (cyber-toad) [reply data/strings/strings.txt:25388]: Strings are fixed in regenerated.
But there is a failure in iOS build: 
`/Users/runner/work/organicmaps/organicmaps/iphone/Maps/UI/Editor/MWMEditorNotesFooter.m:25:30: error: string literal must be prefixed by '@'`
Probably there is some typo in your suggestion for iOS change here: https://github.com/organicmaps/organicmaps/pull/4844#issuecomment-1482503908 
?

Comment (biodranik) [reply data/strings/strings.txt:25388]: 🤦 yes, of course: `L(@"string")`

Comment (cyber-toad) [reply data/strings/strings.txt:25388]: Thanks, I fixed it and code compiles now: https://github.com/cyber-toad/organicmaps/actions/runs/4526319754

**Review by biodranik — APPROVED** [review]
Thank you for the contribution, and welcome to the club!

**Pull request merged**